website/blog/2023-05-02-release-0.15.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Release Notes - Podman Desktop 0.15
+description: Podman Desktop 0.15 has been released! 
+slug: podman-desktop-release-0.15
+authors: [deboer]
+tags: [podman-desktop, release, kubernetes, kind]
+hide_table_of_contents: false
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 0.15 - Cleanliness is next to Podliness!
+
+It has only been two weeks since our last release, but we really wanted to complete a few scenarios,
+fix a few bugs, and show off several design updates and UI improvements that we have been working on.
+
+<!--Main Features-->
+
+- **Podman Version**: Podman 4.5 now included in Windows and Mac installers.
+- **Kind Ingress**: Configure an ingress controller.
+- **Podliness**: Ability to define ports when creating a pod.
+- **Cleanliness**: New navigation bar, dialog, and palette update.
+- **UX and UI Improvements**: Improvements to **Settings** and markdown support.
+
+Podman Desktop 0.15 is now available. [Click here to download it](/downloads)!
+
+![Podman-desktop-0-15-hero](img/podman-desktop-release-0.15/podman-desktop-release-0.15.webp)
+
+<!--truncate-->
+_________________
+
+## Release Details
+
+### Update to Podman v4.5
+
+Podman Desktop 0.15 embeds [Podman 4.5](https://github.com/containers/podman/releases/tag/v4.5) in
+Windows and macOS installers [#2115](https://github.com/containers/podman-desktop/issues/2115).
+
+### Kind Ingress
+
+Wait a minute, didn't we enable this last release? Well yes, last time we did add support for installing
```

Comment:
I felt like we needed to mention the ingress controller because we had a very similar item listed in the previous release. I updated the wording though to be correct and a little clearer the difference between the two.